keps/sig-node/3085-pod-conditions-for-starting-completition-of-sandbox-creation/kep.yaml

```diff
@@ -0,0 +1,51 @@
+title: KEP Template
+kep-number: 3085
+authors:
+  - "@ddebroy"
+  - "@agamdua"
+owning-sig: sig-node
+participating-sigs:
+status: provisional
+creation-date: 2021-12-14
+reviewers:
+  - "@ehashman"
+approvers:
+  - "@derekwaynecarr"
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
```

Comment:
I have to review this one anyways, please set me as PRR approver

Reply:
done